proto/store/audit_log.proto

```diff
@@ -46,3 +51,7 @@ message AuditLog {
     EMERGENCY = 8;
   }
 }
+
+message AuditData {
```

Comment:
We don't need to do double conversion. Let's shelf v1 audit data in this store Any field.